ecs-cli/Gopkg.toml

```diff
@@ -25,7 +25,7 @@
 
 [[constraint]]
   name = "github.com/aws/aws-sdk-go"
-  version = "1.15.2"
+  version = "=1.18.0"
```

Comment:
Why is there an = sign?

Reply:
Good question! @SoManyHs and I recently found out that if you don't have an '=' sign,then `dep ensure` will upgrade the version of the dependency up to the next major version. To pin a version number we should use '='.

> When you specify a version without an operator, dep automatically uses the ^ operator by default. 

https://golang.github.io/dep/docs/Gopkg.toml.html#version

Reply:
Interesting, good to know 👍